install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -64,24 +64,24 @@ datasources:
   datasources.yaml:
     apiVersion: 1
     datasources:
-    - name: Prometheus
```

Comment:
was this failing the linter? I think we use this indentation style pretty often

Reply:
Yes, it faild under current rules.